ODIN_II/SRC/partial_map.cpp

```diff
@@ -186,13 +186,22 @@ void partial_map_node(nnode_t *node, short traverse_number, netlist_t *netlist)
 			break;
 
 		case ADD:
-			if (hard_adders && node->bit_width >= min_threshold_adder){
-				// Check if the size of this adder is greater than the hard vs soft logic threshold
-					instantiate_hard_adder(node, traverse_number, netlist);
-			}else{
-				instantiate_add_w_carry(node, traverse_number, netlist);
+			if (true == mixer->mixHardBlocksOfType(HardBlocksOptimizationTypesEnum::MULTIPLIERS) ||
```

Comment:
Don't check true on a boolean make it your variable and function call explicit about them returning a true.

Don't use camel case, the only place camel case is used is in the parse AST, try to match the style of the file

Is the namespace necessary? I feel it adds complexity by adding more strict and enums to track. Could you use the globally declared operation_list?

Reply:
I will present better phrasing in my next commits, tell me if you agree with changes